BootloaderCorePkg/Tools/CommonUtility.py

```diff
@@ -201,6 +201,12 @@ def get_openssl_path ():
                     openssl_cfg = "C:\\Openssl\\openssl.cfg"
                     if os.path.exists(openssl_cfg):
                         os.environ['OPENSSL_CONF'] = openssl_cfg
+    else :
+        if 'OPENSSL_PATH' not in os.environ:

```

Comment:
why do we do this ?  
This might not work in many Linux env.   openssl might be on some other place on the PATH env, but not necessary on /usr/bin.

Reply:
It tried to fix the build issue where openssl is not in the PATH I think.
Maybe check if os.path.join(os.environ.get ('OPENSSL_PATH', ''), 'openssl') works or not, then check other cases?

Reply:
When openssl is installed in Linux, it will be added into PATH.    I don't think we should do this change.  It will break more things.

Reply:
it looks this could be fixed from signing script. so just close this one.